### ISSUE TITLE
Localization (l10n) scheme for wp1 table and log templates

### ISSUE THREAD

**Issue by audiodude** [openzim/wp1#264]
This affects all of the files in [the templates directory](https://github.com/openzim/wp1/tree/main/wp1/templates).

For Arabic Wikipedia, the tables need to read right-to-left. It's unclear if the Mediawiki software takes care of this automatically or if we need to specify the columns in a different order in the wiki markup.

For other files, things like [[was renamed to]] need to be localized. We should use a proper localization scheme instead of trying to fork these files and use different ones for different languages, because it would soon get unwieldy to keep track of them all.

**Comment by audiodude** [comment]
@alanajjar Do you know if Mediawiki automatically transposes tables into right-to-left for Arabic?

**Comment by Meno25** [comment]
> @alanajjar Do you know if Mediawiki automatically transposes tables into right-to-left for Arabic?

Yes, it does. On wikis that are configured to use an RTL language (such as Arabic) as the content language, MediaWiki automatically displays tables correctly aligned. This includes also the text inside the table.
